README.md

```diff
@@ -24,3 +24,13 @@ $ python src/train.py --model neuralbow trained_models \
     resources/data/python/final/jsonl/valid \
     resources/data/python/final/jsonl/test
 ```
+
+### AST parsing
+
+For AST parsing we use [tree-sitter](https://github.com/tree-sitter/tree-sitter).
+To work with parsers, do the following:
+
+1. Clone the appropriate tree-sitter parser in `src/parsing/tree-sitter-languages`
+2. Add information about it to [setup_tree_sitter.py](src/processing/setup_tree_sitter.py)
+3. Add the language to [utils.py](src/processing/utils.py)
```

Comment:
Not sure what this step means, since there is nothing language specific in `utils.py`. Maybe you've meant `run.py`?

Reply:
This step is outdated after the unification parsing API, thanks